examples/splinterdb_custom_ipv4_addr_sortcmp_example.c

```diff
@@ -259,6 +259,11 @@ ip4_ipaddr_keycmp(const char  *key1,
                   const char  *key2,
                   const size_t key2_len)
 {
+   // Deal with the case of initial keys, where one of the pair maybe uninit'ed
+   if ((key1_len == 0) || (key2_len == 0)) {
+      return KEYCMP_RV_KEY1_EQ_KEY2;
```

Comment:
**Reviewers: NOTE**: - Let's discuss: This seems to be the fix required to deal with the debug assertion failure.

Reply:
Prior to this change, [CI debug quick-tests failed with an assertion](https://runway-ci.eng.vmware.com/builds/9920756).